bootcamp/services/title_service.py

```diff
@@ -25,3 +26,29 @@ def get_by_id(self, title_id):
     def check_duplicates(self, entity):
         title = yield self.get_by_id(entity.title_id)
         raise Return(title is not None)
+
+    @coroutine
+    def add_tag(self, title_uuid, tag_uuid, add):
+        title_uuid = is_valid_uuid_string(title_uuid)
+        tag_uuid = is_valid_uuid_string(tag_uuid)
+
+        title = yield self.store.get(title_uuid)
+
+        tag_list = list(title.tags)
```

Comment:
If title.tags is `None`, then this line will break.
I would suggest to apply a default value to `tags` column in `bootcamp/models/title.py`.
``` python
tags = Column(ARRAY(UUID), default=[], nullable=False)
```